packages/teleport/src/services/auth/makeMfa.ts

```diff
@@ -72,3 +72,61 @@ export function makeMfaAuthenticateChallenge(json): MfaAuthenticateChallenge {
     webauthnPublicKey: webauthnPublicKey,
   };
 }
+
+// makeNavCredentialsCreateResponse takes response from navigator.credentials.create
+// and creates a credential object expected by the server with ArrayBuffer
+// fields converted to Base64URL:
+// - rawId
+// - response.attestationObject
+// - response.clientDataJSON
+export function makeNavCredentialsCreateResponse(res) {
+  // Response can be null if no Credential object can be created.
```

Comment:
it's unclear how these null responses happen, but looking through these comments https://github.com/w3c/webauthn/issues/876, it's rare to happen. Yassine was able to trigger it for `navigator.credentials.get` which was resolved by trying again, but we don't remember _how_ it happened and couldn't reproduce it.

Reply:
If I'm understanding correctly, if we add logic for a single retry on null response [here](https://github.com/gravitational/webapps/blob/lisa/define-webauthn-methods/packages/teleport/src/services/auth/auth.ts#L169), that should solve all but the rarest cases until the underlying webauthn implementation is fixed.